src/osd/OSD.cc

```diff
@@ -6205,7 +6205,7 @@ COMMAND("dump_pg_recovery_stats", "dump pg recovery statistics",
 	"osd", "r", "cli,rest")
 COMMAND("reset_pg_recovery_stats", "reset pg recovery statistics",
 	"osd", "rw", "cli,rest")
-COMMAND("compact", "compact omap",
+COMMAND("compact", "compact object store's omap. WARNING: Compaction probably slows your requests",
```

Comment:
better off squashing this commit into the one which introduces this command. and please wrap at 79 chars.